### DISCUSSION TITLE
Selecting a license when the upstream offers multiple choices

### DISCUSSION THREAD

**Discussion by icp1994** [void-linux/void-packages#45548] [Q&A]
I have seem quite a few rust (`build_style=cargo`) templates which lists `license="Apache-2.0, MIT"` while the corresponding upstreams clearly mentions choosing either "at your option". Since "MIT" licenses require an additional `vlicense`, is there are a reason why we can't set `license="Apache-2.0"` and remove those `vlicense` calls in `post_install()`?

**Comment by classabbyamp** [comment]
really we should have the [spdx operators `AND` `OR` `WITH`](https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d4-composite-license-expressions)

**Comment by icp1994** [comment]
Probably makes sense for AND & WITH but why would downstream distributors (like Void) need to bundle license files with binaries while the upstream licensing policy doesn't require it?

**Comment by Chocimier** [comment]
It is nice to pass through dual-licence choice to users of Void-built binaries, and carrying single file is low enough effort to enable this.

**Comment by icp1994** [comment]
Fair enough.
